Doc/library/ctypes.rst

```diff
@@ -2083,6 +2109,21 @@ Utility functions
    .. availability:: Windows
 
 
+.. function:: dllist()
+   :module: ctypes.util
+
+   Try to provide a list of paths of the shared libraries loaded into the current
+   process.  These paths are not normalized or processed in any way. If the
+   function is not able to determine the list of loaded libraries (either because
+   the current platform is not Windows, MacOS, or Linux and does not support the
+   the ``dl_iterate_phdr`` API in libc, or because an error occurred), it
+   returns ``None``.
+
+   The exact functionality is system dependent.
+
```

Comment:
Please add `Availability` listing the platforms; ISTM should be
```
.. availability:: Windows, macOS, glibc, BSD libc
```
and perhaps `musl` too. The list is at `Doc/tools/extensions/availability.py`.

Reply:
I listed those you did. I _believe_ it would also work as-implemented on iOS, Android, and musl, but I have not verified these myself

Reply:
Testing in an alpine docker image (musl), the code here works _except_ that `find_library('c')` returns none there.
I can still get a handle on libc by using `ctypes.CDLL(None)` -- is this legit to do in the `util` code? 

I see it being used once in the ctypes documentation, but it seems absent from the existing cpython codebase.